src/analyzer.ts

```diff
@@ -259,32 +300,50 @@ export class StreamResolver implements Resolver {
 
   resolveDeferredFile(filePath: string, file: File): void {
     let deferred = this.deferredFiles.get(filePath);
-    deferred.resolve(file.contents.toString());
+    deferred(file.contents.toString());
     this.deferredFiles.delete(filePath);
   }
 
-  accept(url: string, deferred: Deferred<string>): boolean {
-    logger.debug(`accept: ${url}`);
+  canLoad(url: string): boolean {
+    return true;
```

Comment:
We want to match with external URLs so that we can resolve them as empty strings (otherwise we would error). In the future we'll probably want to refactor this into two loaders: 1 that handles external files and 1 that handles local. I'll add a TODO